source/sc/2.4.4.html.md.erb

```diff
@@ -4,27 +4,80 @@ title: 2.4.4 Link Purpose (In Context) (A)
 
 # <%= current_page.data.title %>
 
-[Understanding <%= current_page.data.title %>](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html)
+## WCAG says:
 
-The purpose of every link must be clear from its link text, or its link text plus associated content if assistive technologies recognise the association. This ensures that screen reader users can understand the purpose of links without reading nearby content, and that speech recognition users can target links accurately using voice commands.
+> "The purpose of each link can be determined from the link text alone or from the link text together with its programmatically determined link context" (with one exception)
 
-## Requirements / What to do?
+[Understanding 2.4.4 Link Purpose (In Context)](https://www.w3.org/WAI/WCAG22/Understanding/link-purpose-in-context)
 
-*   Link text clearly indicates the purpose of the link;
-*   Multiple links that point to the same destination have the same link text;
-*   Links have accessible names.
+## What this means
+
+Every link must have text which describes its purpose within its immediate context. The text does not have to be visible, but it must be conveyed to assistive technologies.
+
+## Why it matters
+
+This makes sure that screen reader users can understand the purpose of links in their context, and that speech recognition users can target links accurately using voice commands.
+
+## What to check
+
+The accessible name is the name used by assistive technologies such as screen readers and voice control tools. To find the accessible name of each link, you can use:
+
+* your browser's developer tools
+* a screen reader
+
+Check that each link describes the link's purpose. If the link text itself is not enough to understand the purpose, then also check its immediate context, such as the same paragraph, list item, table cell or an associated table header.
+
+Automated tools such as Axe can detect links with no text, but they cannot determine whether link text is appropriate when it exists.
+
+[How to test in detail for 2.4.4 Link Purpose (in context)](https://github.com/alphagov/wcag-primer/wiki/2.4.4)
+
+## Good example
+
+### Links with unique accessible names
+
+The following screenshot includes three "Change" links related to a fictional person's name, date of birth and customer reference.
+
+<figure>
+    <%= image_tag "../images/244/244-pass-change-links.png", { :alt => 'Change links' } %>
+    <figcaption>Personal details with links to change the user's name, date of birth or customer reference.</figcaption>
+</figure>
+
+As the word "Change" alone is not enough to describe the purpose of each link, additional text has been added for assistive technology users. For example, the first link has an accessible name of "Change name".
```

Comment:
This would pass In Context without the need for hidden content as the links are within a construct (`dt` - `dd`) which programmatically links them to the key-value pairing.